addons/resource/models/resource_calendar_attendance.py

```diff
@@ -29,19 +29,25 @@ class ResourceCalendarAttendance(models.Model):
     # For the hour duration, the compute function is used to compute the value
     # unambiguously, while the duration in days is computed for the default
     # value based on the day_period but can be manually overridden.
-    duration_hours = fields.Float(compute='_compute_duration_hours', string='Duration (hours)')
+    duration_hours = fields.Float(compute='_compute_duration_hours', inverse='_inverse_duration_hours',
```

Comment:
We'll need a migration script to initialize the column and compute the values in SQL to avoid ORM overhead during upgrade.